packages/thumbprint-react/CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## Unreleased
 
+### Changed
+
+-   [Patch] Simplify `Image` component by remove placeholder `div` and CSS positing so the component behaves more predictably. Fixes double download bug in Edge 18.
+-   [Major] Remove `object-fit` CSS from `Image` that was previously being added when the `containerAspectRatio` was present. This prop now only adds placeholder spacing on the `<img>` which is removed onload.
```

Comment:
```suggestion
-   [Major] Only include `object-fit` CSS in `Image` when `height` is provided. That CSS was previously being added when the `containerAspectRatio` was present. `containerAspectRatio` now only adds placeholder spacing on the `<img>` which is removed `onload`.
```

Changed it to be a bit more clear that object-fit CSS sometimes is still included.